pkg/scheduler/plugins/gang/gang.go

```diff
@@ -197,18 +196,6 @@ func (gp *gangPlugin) OnSessionClose(ssn *framework.Session) {
 				klog.Errorf("Failed to update job <%s/%s> condition: %v",
 					job.Namespace, job.Name, err)
 			}
-
-			// allocated task should follow the job fit error
```

Comment:
Why this removed?

Reply:
It is to set those tasks have no fiterror, which have been alocated. It is meaningless. This message can be found in podgroup message and event. After removing it , those pods will have a message as the pictures show that `it can be assigned at node xxx`.